Python/dp/Rod_cutting_dp.py

```diff
@@ -0,0 +1,48 @@
+'''
+Cutting a Rod with Maximum value
+Author: Sakar Mehra (@sakar97)
+Problem Statement: Given a rod of length n inches and an array of prices that
+        contains prices of all pieces of size smaller than n.
+        Determine the value obtain by cutting up the rod and selling the pieces.
+
+        For Example:
+            length   | 1   2   3   4   5   6   7   8
+            --------------------------------------------
+            price    | 1   5   8   9  10  17  17  20
+    Output: 22
+
+    Time Complexity: O(N^2)
+'''
+# A Dynamic Programming solution for Rod cutting problem
+INT_MIN = -32767
+
+# Returns the best obtainable price for a rod of length n and
+# price[] as prices of different pieces
+
+
+def cutRod(price, n):
```

Comment:
what does the variable `n` mean here? It be better if the variable name is something which gives a context of what it is supposed to store

Reply:
N is simply the size of the rod.

Reply:
Yeah i meant you should change it